client/docs/swagger-ui/swagger.yaml

```diff
@@ -41,10 +41,10 @@ paths:
                       format: byte
       tags:
         - Query
-  /seda-chain/wasm-storage/data_request_wasm/{hash}:
+  /seda-chain/wasm-storage/executor_wasm/{hash}:
```

Comment:
`oracle_program` no? They aren't specific to execution since they can also be used for tallying.

Unless these are the executor WASMs for the overlay to use? Maybe we could update the description to say it's for overlay use?

Reply:
I think this is just git diff detecting wrong lines